docs/getting-started/introduction.md

```diff
@@ -1,121 +1,223 @@
-completely open-source, modular, and extensible, allowing it to be fully tailored to the requirements of any project.
-Below are the key features of the platform:
+:::tip Directus Vocabulary
 
-- **No-Code App for anyone to manage data**
-- **Data Connection Toolkit with REST & GraphQL**
-- **Supports Any SQL Database**
-- **Authorization & Caching**
-- **Completely Free & Open Source**
+In order keep things simpler, _especially for the no-code users_, Directus uses friendlier names for many database terms
+and technical concepts, including [Project](/getting-started/glossary/#projects) (database),
+[Collection](/getting-started/glossary/#collections) (table), [Field](/getting-started/glossary/#fields) (column),
+[Item](/getting-started/glossary/#items) (record), and [Type](/getting-started/glossary/#types) (datatype).
 
-::: tip What's in a name?
+<!--
+@TODO getting-started > learn-directus
+For more information on Directus-specific terms, see the guide on [how to learn Directus](/getting-started/learn-directus).
+-->
 
-"Directus" ([duh REKT iss](http://audio.pronouncekiwi.com/Salli/Directus)) is Latin for: _laid straight, arranged in
-lines_. The broadest goal of our platform is to present data in a simple, orderly, and intuitive way.
+:::
+
+<!--
+:::tip Ready to see what Directus can do?
+
+Checkout the core features and learn Directus hands-on in the [Quickstart Guide](/getting-started/quickstart/).
+
+@TODO getting-started > crashcourse
+Change this CTA to the 20-30 min crashcourse
+
+:::
+-->
+
+## When to Use It
+
+**Directus can power any data-driven project and can be linked or removed at any time.**
+
+You'll be able to build, monitor or manage any data model or app you desire: IoT fleets, ecommerce, SaaS, business data
+analytics, multi-channel content, _or anything else_.... If it involves SQL and digital files, you can use Directus.
+With this in mind, four broad use cases emerge:
+
+- **Backend as a Service**\
+  An end-to-end data solution. Efficiently connects data, auto-generates exhaustive APIs for projects that scale, provides
+  webhooks and task automation. Completely detached from the database, you're free to link and remove Directus anytime or
+  use any other service in tandem.
+- **Headless CMS**\
+  Remember: _content_ is just data from a database! Manage any omnichannel digital experience. Deliver file assets and data
+  across websites, apps, kiosks, digital signage... _the sky's the limit!_
+- **Internal Tool Builder**\
+  The whole team can build custom apps! Ditch the spreadsheet and quickly build back-office apps and admin panels for customers,
+  inventory, projects, marketing _or anything else._
+- **Data Management and Analytics**\
+  Establish _a single source of truth_ for all data. Build no-code analytics dashboards to gain insights into company KPIs
+  and other metrics. Coalesce previously siloed department data.
+
+<!--
+mention SaaS in there
```

Comment:
@benhaynes you listed this as a buzzword to add, tbh I'm not sure what you meant. (I know what SaaS, but idk how you wanted me to work it in... Directus as SaaS, Directus as a way to manage other SaaS, or something other???